scylla-cql/src/frame/value.rs

```diff
@@ -866,10 +867,26 @@ macro_rules! impl_value_list_for_map {
     };
 }
 
-impl_value_list_for_map!(HashMap, String);
-impl_value_list_for_map!(HashMap, &str);
-impl_value_list_for_map!(BTreeMap, String);
-impl_value_list_for_map!(BTreeMap, &str);
+// Implement ValueList for maps, which serializes named values
+macro_rules! impl_value_list_for_hash_map {
+    ($map_type:ident, $key_type:ty) => {
+        impl<T: Value, S: BuildHasher> ValueList for $map_type<$key_type, T, S> {
```

Comment:
Nit: the `map_type` argument is now not needed, right? It should always be `HashMap`.

Reply:
Agree.

Reply:
Updated.